types/confirmdialog/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
For new packages, it should only extends "dtslint/dt.json", no other rules.

Reply:
it is done

Reply:
"Since the 2 packages are created by you and they already bundle types, so you don't have to submit their types to this repository."

I don't unsterstand what you means. There was not package before with that 2 Definitions. or not?
I have not see that